app/models/concerns/delegate_attribute.rb

```diff
@@ -0,0 +1,54 @@
+module DelegateAttribute
```

Comment:
ah, okay, forgot that test_track is open source. Question for @Betterment/test_track_core. In this case, do we prefer to just use `delegate` rather than `delegate_attribute`
```ruby
delegate :hypothesis, :hypothesis=, :assignment_criteria, :assignment_criteria=, :description, :description=, :owner, :owner=, :location, :location=, :platform, :platform=, to: :split
```